File: C++17/백준/Bronze/10833. 사과/사과.cc
#include <iostream>
using namespace std;

int main () {
    int n;
    cin >> n;
    
    int total_apls = 0;
    
    for (int i = 0; i < n; i++) {
        int apls, snts;
        cin >> apls >> snts;
        
        total_apls += snts % apls;
    }
    
    cout << total_apls;
    
    return 0;
}

File: Java/백준/Bronze/10833. 사과/사과.java
import java.util.Scanner;

public class Main {
    public static void main (String[] args) {
        Scanner sc = new Scanner (System.in);
        
        int n = sc.nextInt();
        
        int total_apls = 0;
        
        for (int i = 0; i < n; i++) {
            int snts = sc.nextInt();
            int apls = sc.nextInt();
            
            total_apls += (apls % snts);
        }
        
        System.out.println(total_apls);
        
        sc.close();
    }
}